src/cbmc/cbmc_parse_options.cpp

```diff
@@ -662,6 +662,8 @@ int cbmc_parse_optionst::get_goto_program(
     if(!binaries.empty())
       config.set_from_symbol_table(symbol_table);
 
+    config.set_addr_bits(cmdline, symbol_table);
```

Comment:
I believe this should be done by `configt` automatically, as doing this only in "cbmc_parse_options.cpp" may not be sufficient.